test/format/random.c

```diff
@@ -76,7 +76,7 @@ random_kv(void *arg)
         wt_wrap_open_cursor(session, table->uri, config, &cursor);
 
         /* This is just a smoke-test, get some key/value pairs. */
-        for (i = mmrand(&g.extra_rnd, 0, WT_THOUSAND); i > 0; --i) {
+        for (i = mmrand(&g.extra_rnd, 0, WT_THOUSAND); i > 0 && !g.workers_finished; --i) {
```

Comment:
I would like to check some pre-existing logic: `mmrand` may initialize `i=0` resulting in no key retrieval. Under the guise of a test this seems "fair enough", but I want to confirm it matches expectations.

Reply:
Really good question!